<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenTab</class>
 <widget class="QWidget" name="ScreenTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Media List:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="mediaListTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="addMediaPushButton">
           <property name="text">
            <string>Add Media</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addExtensionPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add Extension</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeRowPushButton">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveRowUpPushButton">
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/arrow-090.png</normaloff>:/icons/arrow-090.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveRowDownPushButton">
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/arrow-270.png</normaloff>:/icons/arrow-270.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Current Video:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaPreview" name="mediaPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="randomMediaLabel">
           <property name="text">
            <string>Rondom Media:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="randomMediaCheckBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="loadListPushButton">
           <property name="text">
            <string>Load Screen</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/folder-open-document.png</normaloff>:/icons/folder-open-document.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveListPushButton">
           <property name="text">
            <string>Save Screen</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/disk.png</normaloff>:/icons/disk.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startPlayingPushButton">
           <property name="toolTip">
            <string>Start or stop playing (Ctrl+R)</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/control.png</normaloff>:/icons/control.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddMedia">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Media</string>
   </property>
   <property name="toolTip">
    <string>Add Media</string>
   </property>
  </action>
  <action name="actionRemoveRow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Row</string>
   </property>
  </action>
  <action name="actionMoveMediaUp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/arrow-090.png</normaloff>:/icons/arrow-090.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMoveMediaDown">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/arrow-270.png</normaloff>:/icons/arrow-270.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/disk-black.png</normaloff>:/icons/disk-black.png</iconset>
   </property>
   <property name="text">
    <string>Save Screen</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/folder-open-document.png</normaloff>:/icons/folder-open-document.png</iconset>
   </property>
   <property name="text">
    <string>Load Screen</string>
   </property>
  </action>
  <action name="actionAddExtension">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Extension</string>
   </property>
   <property name="toolTip">
    <string>Add Extension</string>
   </property>
  </action>
  <action name="actionStartStopPlaying">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/control.png</normaloff>:/icons/control.png</iconset>
   </property>
   <property name="text">
    <string>Start playing</string>
   </property>
   <property name="toolTip">
    <string>actionStartStopPlaying</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaPreview</class>
   <extends>QGraphicsView</extends>
   <header>mediapreview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addMediaPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAddMedia</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeRowPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionRemoveRow</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveRowDownPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionMoveMediaDown</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveRowUpPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionMoveMediaUp</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveListPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSave</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadListPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionLoad</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addExtensionPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAddExtension</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startPlayingPushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionStartStopPlaying</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
